apps/e2e/helpers/table.ts

```diff
@@ -57,7 +57,7 @@ export function getInventoryTable(parent: DashboardNode, view: InventoryTableVie
 
 const defaultInventoryRowValues: InventoryRowValues = {
 	isbn: "",
-	price: "N/A",
+	price: "€0.00",
 	quantity: 1,
```

Comment:
I think this should be done at query level (`COALESCE(book.price, 0) AS price`).